### ISSUE TITLE
capistrano-maintenace breaks 'cap' binstubs calls 

### ISSUE THREAD

**Issue by lustremedia** [capistrano/maintenance#27]
cap -T
cap aborted!
LoadError: cannot load such file -- capistrano/maintenance
Capfile:26:in `<top (required)>'
.rbenv/versions/2.0.0-p353/bin/cap:23:in `load'
.rbenv/versions/2.0.0-p353/bin/cap:23:in `<main>'
(See full trace by running task with --trace)

Using:
rbenv
capistrano 3
ruby2
rails

Before installing the gem it worked without a problem.

**Comment by gsusmonzon** [comment]
I had same problem. It seems that capistrano was requiring the wrong gem. I could not tame rbenv , bundler , capistrano or whatever.

I resort to
1)  copy the `maintenance.rake` file of this gem ( https://github.com/capistrano/maintenance/blob/master/lib/capistrano/tasks/maintenance.rake )  to the project ./lib/capistrano/tasks
2) copy the template/maintenance.html   to the project ./lib/capistrano/tasks/templates
3) ensure this rake task is included in your Capfile (you should have something like this at the end)
```ruby
# Loads custom tasks from `lib/capistrano/tasks' if you have any defined.
Dir.glob('lib/capistrano/tasks/*.rake').each { |r| import r }
```
4) remove references to this gem in your project 'Capfile'

5) remove references in the Gemfile and do a `bundle install`

It worked for me

**Comment by kmeister2000** [comment]
I also ran into the problem and saw that there's a simple fix already in the master branch:
https://github.com/capistrano/maintenance/commit/9c927e851506fadd8fee2eaef2645b4248d68c9e

Simply add require: false to the line in your GemFile and it fixes the problem - well, at least for me.  Ruby 2.2.3, Cap 3, Rails 4.1

**Comment by mattbrictson** [comment]
Fixed via #32, I think.
